get_edx_platform.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+ROOT=$(pwd)
+REPO="${OPENEDX_REPO:-https://github.com/edx/edx-platform}"
+BRANCH="${OPENEDX_RELEASE:-master}"
+WORKING_COPY="${ROOT}/.tox/edx-platform"
+
+function check_repo_remote() {
+    (cd "$WORKING_COPY" && git remote -v | grep "$REPO")
+}
+
+function clone() {
+    echo "Cloning edx-platform from $REPO, branch $BRANCH"
+    rm -rf "$WORKING_COPY"
+    git clone -b "$BRANCH" "$REPO" "$WORKING_COPY"
```

Comment:
Comment from https://github.com/open-craft/edx-learndot/pull/2#discussion_r174325783 applies here too.

Reply:
Added support for cloning from a local reference repo.